.versionbump.cfg

```diff
@@ -0,0 +1,16 @@
+[bumpversion]
+current_version = 0.0.3
+tag = False
```

Comment:
You will also need to add `bumpversion` to the dev dependencies. Otherwise the release pipeline will not install it.